scipy/stats/_morestats.py

```diff
@@ -1250,6 +1250,12 @@ def boxcox_normmax(
     6.000...
     """
     x = np.asarray(x)
+
+    if np.any(~np.isfinite(x) | (x < 0)):
```

Comment:
is the `~` idiomatic here? Fine if so, but I haven't seen it before

Reply:
[Not uncommon](https://stackoverflow.com/questions/13728708/inverting-a-numpy-boolean-array-using), I don't think. What do you prefer? I used to use `np.logical_not`, but this is a lot shorter.

Reply:
True, that is a lot less verbose, fine with me

Reply:
I could rephrase as `not np.all(..)` and it would probably be a bit more efficient, avoiding an array operation.